examples/ex_07_save_load_model.nim

```diff
@@ -0,0 +1,93 @@
+import arraymancer, strformat, os
+
+discard """
```

Comment:
Instead of using `discard """` you can just either use double `##` or straight `#[` and `]#` to open and close a multi line comment.
This has the advantage that it's a proper documentation comment and would show up if running the file through `nim doc` (even though that's not important for these examples).

Reply:
Thanks @Vindaar I've replaced it with a simple multiline comment

Reply:
@Vindaar I've also added the example to the .nimble file